frontends/web/src/api/devicessync.ts

```diff
@@ -77,3 +79,24 @@ export const attestationCheckDone = (
   });
   return unsubscribe;
 };
+
+
+export const signProgress = (
+  cb: (meta: TSignProgress) => void
+): TUnsubscribe => {
+  return apiWebsocket(payload => {
```

Comment:
just use subscribeLegacy like the other functions in this module do.

have a look at frontends/web/src/utils/event-legacy.ts there it uses apiWebsocket internally